chirp/drivers/tdh8.py

```diff
@@ -1315,6 +1302,22 @@ def set_memory(self, mem):
             elif block_xy[1] == 8:
                 _usedflags.block8 = 0
 
+    def set_memory(self, mem):
+        _mem = self._get_mem(mem.number)
+        _nam = self._get_nam(mem.number)
+ 
+        self.evaluate_usedflags(mem)
+
+        if mem.empty:
+            _mem.set_raw("\xff" * 16)
+            if self.MODEL in H8_LIST:
+                _nam.set_raw("\xff" * 16)
```

Comment:
And if the structures are right, `fill_raw()` here will avoid the need for the conditional.